### ISSUE TITLE
Store MRI metadata in a DataFrame during hd5 production

### ISSUE THREAD

**Issue by ndiamant** [broadinstitute/ml4h#405]
**What**
Save all of the tabular data (age, height, weight, slice metadata, etc.) from Dicoms into a manifest data frame during tensorization.
Consider not saving the tabular data in the hd5s.

**Why**
It will be much easier to explore the tabular data if it's in a csv.
We will also save a lot of new tabular data.
The hd5s will be less cluttered if they don't contain the tabular data.

**How**
`pydicom` reads metadata. 
We can put the metadata for each slice into a dictionary (collapsing values that do not change over slices).
Then in [write_tensors](https://github.com/broadinstitute/ml4h/blob/6fc20fc39e139a4d27ed42837a418a11b747c3b2/ml4h/tensorize/tensor_writer_ukbb.py#L81) we can aggregate the dictionaries into a data frame.

**Acceptance Criteria**
A csv with tabular data from dicoms is output during dicom tensorization.

**Comment by mklarqvist** [comment]
Example MRI DICOM meta data

```
Dataset.file_meta -------------------------------
(0002, 0000) File Meta Information Group Length  UL: 214
(0002, 0001) File Meta Information Version       OB: b'\x00\x01'
(0002, 0002) Media Storage SOP Class UID         UI: MR Image Storage
(0002, 0003) Media Storage SOP Instance UID      UI: <HIDDEN>
(0002, 0010) Transfer Syntax UID                 UI: Explicit VR Little Endian
(0002, 0012) Implementation Class UID            UI: <HIDDEN>
(0002, 0013) Implementation Version Name         SH: 'OFFIS_DCMBP_360'
(0002, 0016) Source Application Entity Title     AE: 'AWP141243'
-------------------------------------------------
(0008, 0005) Specific Character Set              CS: 'ISO_IR 100'
(0008, 0008) Image Type                          CS: ['ORIGINAL', 'PRIMARY', 'M', 'DIS2D']
(0008, 0012) Instance Creation Date              DA: '20170718'
(0008, 0013) Instance Creation Time              TM: '094203.128000'
(0008, 0016) SOP Class UID                       UI: MR Image Storage
(0008, 0018) SOP Instance UID                    UI: <HIDDEN>
(0008, 0020) Study Date                          DA: '20170718'
(0008, 0021) Series Date                         DA: '20170718'
(0008, 0022) Acquisition Date                    DA: '20170718'
(0008, 0023) Content Date                        DA: '20170718'
(0008, 0030) Study Time                          TM: <HIDDEN>
(0008, 0031) Series Time                         TM: <HIDDEN>
(0008, 0032) Acquisition Time                    TM: '094136.542500'
(0008, 0033) Content Time                        TM: '094203.128000'
(0008, 0050) Accession Number                    SH: ''
(0008, 0060) Modality                            CS: 'MR'
(0008, 0070) Manufacturer                        LO: 'SIEMENS'
(0008, 0080) Institution Name                    LO: 'UK Biobank'
(0008, 0081) Institution Address                 ST: 'Hawick Trading Estate StreetNo,Newcastle upon Tyne,District,GB,NE6 1AS'
(0008, 0090) Referring Physician's Name          PN: ''
(0008, 1010) Station Name                        SH: 'AWP141243'
(0008, 1030) Study Description                   LO: 'UKBB^abdomen'
(0008, 103e) Series Description                  LO: 'Dixon_noBH_in'
(0008, 1040) Institutional Department Name       LO: 'Department'
(0008, 1050) Performing Physician's Name         PN: ''
(0008, 1090) Manufacturer's Model Name           LO: 'Aera'
(0010, 0010) Patient's Name                      PN: <HIDDEN>
(0010, 0020) Patient ID                          LO: <HIDDEN>
(0010, 0030) Patient's Birth Date                DA: <HIDDEN>
(0010, 0040) Patient's Sex                       CS: <HIDDEN>
(0010, 1010) Patient's Age                       AS: <HIDDEN>
(0010, 1020) Patient's Size                      DS: <HIDDEN>
(0010, 1030) Patient's Weight                    DS: <HIDDEN>
(0018, 0015) Body Part Examined                  CS: 'ABDOMEN'
(0018, 0020) Scanning Sequence                   CS: 'GR'
(0018, 0021) Sequence Variant                    CS: ['SP', 'OSP']
(0018, 0022) Scan Options                        CS: 'PFP'
(0018, 0023) MR Acquisition Type                 CS: '3D'
(0018, 0024) Sequence Name                       SH: '*fl3d2'
(0018, 0025) Angio Flag                          CS: 'N'
(0018, 0050) Slice Thickness                     DS: "3.0"
(0018, 0080) Repetition Time                     DS: "6.67"
(0018, 0081) Echo Time                           DS: "4.77"
(0018, 0083) Number of Averages                  DS: "1.0"
(0018, 0084) Imaging Frequency                   DS: "63.67827"
(0018, 0085) Imaged Nucleus                      SH: '1H'
(0018, 0086) Echo Number(s)                      IS: "2"
(0018, 0087) Magnetic Field Strength             DS: "1.5"
(0018, 0089) Number of Phase Encoding Steps      IS: "90"
(0018, 0091) Echo Train Length                   IS: "2"
(0018, 0093) Percent Sampling                    DS: "71.0"
(0018, 0094) Percent Phase Field of View         DS: "75.0"
(0018, 0095) Pixel Bandwidth                     DS: "440.0"
(0018, 1000) Device Serial Number                LO: '141243'
(0018, 1020) Software Versions                   LO: 'syngo MR D13'
(0018, 1030) Protocol Name                       LO: 'Dixon_noBH'
(0018, 1251) Transmit Coil Name                  SH: 'Body'
(0018, 1310) Acquisition Matrix                  US: [224, 0, 0, 119]
(0018, 1312) In-plane Phase Encoding Direction   CS: 'COL'
(0018, 1314) Flip Angle                          DS: "10.0"
(0018, 1315) Variable Flip Angle Flag            CS: 'N'
(0018, 1316) SAR                                 DS: "0.15825931900621"
(0018, 1318) dB/dt                               DS: "0.0"
(0018, 5100) Patient Position                    CS: 'HFS'
(0019, 0010) Private Creator                     LO: 'SIEMENS MR HEADER'
(0019, 1008) [CSA Image Header Type]             CS: 'IMAGE NUM 4'
(0019, 1009) [CSA Image Header Version ??]       LO: '1.0'
(0019, 100b) [SliceMeasurementDuration]          DS: "23897.5"
(0019, 100f) [GradientMode]                      SH: 'Fast'
(0019, 1011) [FlowCompensation]                  SH: 'No'
(0019, 1012) [TablePositionOrigin]               SL: [0, 0, -1183]
(0019, 1013) [ImaAbsTablePosition]               SL: [0, 0, -1213]
(0019, 1014) [ImaRelTablePosition]               IS: [0, 0, -30]
(0019, 1015) [SlicePosition_PCS]                 FD: [-250.00000763, -187.50000572, -49.5]
(0019, 1017) [SliceResolution]                   DS: "1.0"
(0019, 1018) [RealDwellTime]                     IS: "5100"
(0020, 000d) Study Instance UID                  UI: <HIDDEN>
(0020, 000e) Series Instance UID                 UI: <HIDDEN>
(0020, 0010) Study ID                            SH: '1'
(0020, 0011) Series Number                       IS: "1"
(0020, 0012) Acquisition Number                  IS: "1"
(0020, 0013) Instance Number                     IS: "49"
(0020, 0032) Image Position (Patient)            DS: [-250.00000762939, -187.50000572205, -79.5]
(0020, 0037) Image Orientation (Patient)         DS: [1, 0, 0, 0, 1, 0]
(0020, 0052) Frame of Reference UID              UI: <HIDDEN>
(0020, 1040) Position Reference Indicator        LO: ''
(0020, 1041) Slice Location                      DS: "-79.5"
(0028, 0002) Samples per Pixel                   US: 1
(0028, 0004) Photometric Interpretation          CS: 'MONOCHROME2'
(0028, 0010) Rows                                US: 168
(0028, 0011) Columns                             US: 224
(0028, 0030) Pixel Spacing                       DS: [2.2321429252625, 2.2321429252625]
(0028, 0100) Bits Allocated                      US: 16
(0028, 0101) Bits Stored                         US: 12
(0028, 0102) High Bit                            US: 11
(0028, 0103) Pixel Representation                US: 0
(0028, 0106) Smallest Image Pixel Value          US: 0
(0028, 0107) Largest Image Pixel Value           US: 1090
(0028, 1050) Window Center                       DS: "657.0"
(0028, 1051) Window Width                        DS: "1342.0"
(0028, 1055) Window Center & Width Explanation   LO: 'Algo1'
(0029, 0010) Private Creator                     LO: 'SIEMENS CSA HEADER'
(0029, 0011) Private Creator                     LO: 'SIEMENS MEDCOM HEADER2'
(0029, 1008) [CSA Image Header Type]             CS: 'IMAGE NUM 4'
(0029, 1009) [CSA Image Header Version]          LO: '20170718'
(0029, 1010) [CSA Image Header Info]             OB: Array of 10848 elements
(0029, 1018) [CSA Series Header Type]            CS: 'MR'
(0029, 1019) [CSA Series Header Version]         LO: '20170718'
(0029, 1020) [CSA Series Header Info]            OB: Array of 79760 elements
(0029, 1160) [Series Workflow Status]            LO: 'com'
(0032, 1060) Requested Procedure Description     LO: 'UKBB abdomen'
(0040, 0244) Performed Procedure Step Start Date DA: '20170718'
(0040, 0245) Performed Procedure Step Start Time TM: '094016.953000'
(0040, 0253) Performed Procedure Step ID         SH: 'MR20170718094016'
(0040, 0254) Performed Procedure Step Descriptio LO: 'UKBB^abdomen'
(0040, 0280) Comments on the Performed Procedure ST: ''
(0051, 0010) Private Creator                     LO: 'SIEMENS MR HEADER'
(0051, 1008) [CSA Image Header Type]             CS: 'IMAGE NUM 4'
(0051, 1009) [CSA Image Header Version ??]       LO: '1.0'
(0051, 100a) [Unknown]                           LO: 'TA 23.89'
(0051, 100b) [AcquisitionMatrixText]             LO: '119p*224'
(0051, 100c) [Unknown]                           LO: 'FoV 375*500'
(0051, 100d) [Unknown]                           SH: 'SP F79.5'
(0051, 100e) [Unknown]                           LO: 'Tra'
(0051, 100f) [CoilString]                        LO: 'BC'
(0051, 1012) [Unknown]                           SH: 'TP F30'
(0051, 1013) [PositivePCSDirections]             SH: '+LPH'
(0051, 1016) [Unknown]                           LO: 'M/DIS2D'
(0051, 1017) [Unknown]                           SH: 'SL 3.0'
(0051, 1019) [Unknown]                           LO: 'A1/PFP'
(7fe0, 0010) Pixel Data                          OW: Array of 75264 elements
```

**Comment by mklarqvist** [comment]
Addressed for whole-body MRI #425
